evalml/pipelines/components/transformers/scalers/standard_scaler.py

```diff
@@ -32,13 +29,16 @@ def __init__(self, random_seed=0, **kwargs):
 
     def transform(self, X, y=None):
         X = infer_feature_types(X)
-        original_ltypes = X.ww.schema.logical_types
+        original_schema = X.ww.schema
         X = X.ww.select_dtypes(exclude=["datetime"])
         X_t = self._component_obj.transform(X)
         X_t_df = pd.DataFrame(X_t, columns=X.columns, index=X.index)
-        return _retain_custom_types_and_initalize_woodwork(
-            original_ltypes, X_t_df, ltypes_to_ignore=[Integer, Categorical, Boolean]
-        )
+
+        columns = X.ww.select(
+            exclude=[Integer, Categorical, Boolean], return_schema=True
+        ).columns
+        X_t_df.ww.init(schema=original_schema._get_subset_schema(columns))
```

Comment:
same comment here about being able to just use the schema from the `X.ww.select` call and avoid using `_get_subset_schema`.

Question about the exclude logical types (which I see were here before, so this question was probably also relevant before): Are you trying to exclude all columns that are categorical in nature or just ones with the `Categorical` logical type? Because this would end up including columns with a `PostalCode` logical type or one of the others that has the `'category'` semantic tag in the final dataframe. 

If you'd like to exclude all logical types with the `'category'` semantic tag, you can change `Categorical` to `'category'`, and it'll exclude all of those logical types as well.